### PR TITLE
Fabo/cross network ledger

### PR DESCRIPTION
**Pull request by faboweb**
Small refactor (pulled the ledger module out of the store) and setting of the bech32 prefix when requesting addresses from the ledger

Needs manual test

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Signing does not require a human-readable prefix. So the only thing to watch for it that when signing in with the Ledger, the address shows correctly and the queries are done with that address.

**Comment by faboweb** [comment]
Still needs some test coverage

File: tests/unit/specs/components/ActionModal/utils/ActionManager.spec.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ describe("ActionManager", () => {
     }
   })
 
-  describe("simulating and sending", async () => {
+  describe("simulating and sending", () => {
     beforeEach(async () => {
       const context = {
         url: "blah",
```
Comment (jbibla) [inline tests/unit/specs/components/ActionModal/utils/ActionManager.spec.js]: lol

File: config.js [hunk 2]
```diff
@@ -20,5 +20,14 @@ export default {
   graphqlHost: graphql,
 
   e2e: process.env.VUE_APP_E2E || false,
-  enableTxAPI: process.env.VUE_APP_ENABLE_TX_API === "true" || false
+  enableTxAPI: process.env.VUE_APP_ENABLE_TX_API === "true" || false,
+
+  bech32Prefixes: {
```
Comment (faboweb) [inline config.js]: This should be received from the API. I will create an issue.

Comment (faboweb) [reply config.js]: https://github.com/luniehq/lunie/issues/3402

**Comment by jbibla** [comment]
this feature feels like magic.

**Comment by jbibla** [comment]
tracking proposed changes here:

https://github.com/luniehq/cosmos-ledger/issues/21

**Pull request merged**